web/src/main/webapp/dataViews/tableView.ts

```diff
@@ -814,22 +825,22 @@ export class TableView extends TableViewBase implements IScrollTarget {
         if (this.startPosition == null || this.rowCount == null)
             return;
         if (this.rowCount <= 0 || this.dataRowsDisplayed <= 0)
-            // we show everything
+        // we show everything
```

Comment:
the indentation of the comment was correct: it applies to the 'then' branch only.